Puppetfile

```diff
@@ -1,14 +1,29 @@
 forge 'https://forgeapi.puppetlabs.com'
 
-mod 'katello/candlepin',             :git => 'https://github.com/Katello/puppet-candlepin'
-mod 'katello/foreman_proxy_content', :git => 'https://github.com/Katello/puppet-foreman_proxy_content'
-mod 'katello/certs',                 :git => 'https://github.com/Katello/puppet-certs'
-mod 'katello/common',                :git => 'https://github.com/Katello/puppet-common'
-mod 'katello/crane',                 :git => 'https://github.com/Katello/puppet-crane'
-mod 'katello/katello',               :git => 'https://github.com/Katello/puppet-katello'
-mod 'katello/pulp',                  :git => 'https://github.com/Katello/puppet-pulp'
-mod 'katello/qpid',                  :git => 'https://github.com/Katello/puppet-qpid'
-mod 'katello/service_wait',          :git => 'https://github.com/Katello/puppet-service_wait'
+mod 'katello/candlepin',             '>= 2.0.0 < 2.1.0'
+mod 'katello/foreman_proxy_content', '>= 4.0.0 < 4.1.0'
+mod 'katello/certs',                 '>= 1.0.0 < 1.1.0'
+mod 'katello/common',                '>= 1.0.0 < 1.1.0'
+mod 'katello/crane',                 '>= 2.0.0 < 2.1.0'
+mod 'katello/katello',               '>= 6.0.0 < 6.1.0'
+mod 'katello/pulp',                  '>= 4.0.0 < 4.1.0'
+mod 'katello/qpid',                  '>= 2.0.0 < 2.1.0'
+mod 'katello/service_wait',          '>= 1.0.0 < 1.1.0'
 
 # Katello devel specific modules
 mod 'katello/katello_devel',         :git => 'https://github.com/Katello/puppet-katello_devel'
+forge 'https://forgeapi.puppetlabs.com'
+
+# Dependencies
+mod 'puppetlabs/mysql',         '>= 3.8.0'
+mod 'puppetlabs/postgresql',    '>= 4.8.0'
+mod 'puppetlabs/puppetdb'
+mod 'theforeman/dhcp',          :git => 'https://github.com/theforeman/puppet-dhcp'
+mod 'theforeman/dns',           :git => 'https://github.com/theforeman/puppet-dns'
+mod 'theforeman/git',           :git => 'https://github.com/theforeman/puppet-git'
+mod 'theforeman/tftp',          :git => 'https://github.com/theforeman/puppet-tftp'
+
+# Top-level modules
+mod 'theforeman/foreman',       :git => 'https://github.com/theforeman/puppet-foreman'
+mod 'theforeman/foreman_proxy', :git => 'https://github.com/theforeman/puppet-foreman_proxy'
+mod 'theforeman/puppet',        :git => 'https://github.com/theforeman/puppet-puppet'
```

Comment:
one question would be, should this be set somewhere at branch/release time? 

Rather than having to know to pass in the ENV variable every single time it gets run?